### DISCUSSION TITLE
Search not working on github pages

### DISCUSSION THREAD

**Discussion by axjms1** [willfaught/paige#35] [Q&A] [answered]
Hello,

 I am sure this is my problem and not an issue for the actual theme but I can't seem to get the search page working when I deploy to github pages.  I get this error : Cannot load /index.json Response but I do have the toml page configured with this as outputs.

[outputs]
home = ["html", "json", "RSS"]

https://axjms1.github.io/portfolio/search/

**Comment by willfaught** [comment, marked as answer]
Hi, @axjms1! Thanks for reporting this issue. It was indeed a bug. It's now fixed in the latest version: v0.65.2.

**Reply by axjms1** [reply]
What is the best way to update to the latest version?

**Reply by willfaught** [reply]
hugo mod get github.com/willfaught/paige@latest

**Reply by axjms1** [reply]
Thank you!

**Reply by willfaught** [reply]
I added the update info to the readme setup steps.
